vars/signArtifacts.groovy

```diff
@@ -138,7 +138,24 @@ void call(Map args = [:]) {
                """
                 }
         }
-        else {
+        else if (args.platform == 'mac') {
+            withCredentials([usernamePassword(credentialsId: "${GITHUB_BOT_TOKEN_NAME}", usernameVariable: 'GITHUB_USER', passwordVariable: 'GITHUB_TOKEN'),
+                string(credentialsId: 'jenkins-signer-mac-role', variable: 'SIGNER_MAC_ROLE'),
+                string(credentialsId: 'jenkins-signer-mac-external-id', variable: 'SIGNER_MAC_EXTERNAL_ID'),
+                string(credentialsId: 'jenkins-signer-mac-unsigned-bucket', variable: 'SIGNER_MAC_UNSIGNED_BUCKET'),
+                string(credentialsId: 'jenkins-signer-mac-signed-bucket', variable: 'SIGNER_MAC_SIGNED_BUCKET')]) {
+                sh """
+                   #!/bin/bash
```

Comment:
Suggest
```
sh """#!/bin/bash
```

So Jenkins will actually use bash instead of sh.

Reply:
I did not change the default behavior we already have for other signers. Do you still want to change?

Reply:
Ok then, if this works in you test then I am find, it is just you are using `export <>=<>` which is not `sh` but `bash` buildin, so just want to make sure they take effect correctly.